vllm/entrypoints/openai/serving_chat.py

```diff
@@ -83,6 +84,8 @@ def __init__(self,
                 self.tool_parser = MistralToolParser
             elif tool_parser == "hermes":
                 self.tool_parser = Hermes2ProToolParser
+            elif tool_parser == "llama":
```

Comment:
maybe `llama3` or `llama3_json`? it may be desirable to have an option to use the XML-style tool calling + parser in the future, so we should distinguish between the two.

Reply:
I like `llama3_json`. Thanks for the suggestion!